bloom-housing/bloom
# feat: prisma partners listings

# Pull Request Template

## Issue Overview

This PR addresses #3725


## Description

This PR tackles the listings page connection to the new backend on the partners site. It covers all the flows of creating and editing listings. 

It does not cover the following:
- listing and application export is not completed on the backend so that is not included in the PR

There are also a couple of bug fixes found during bug bash:
- Partner user editing the email of a user

## How Can This Be Tested/Reviewed?

After reseeding the prisma backend start up the backend and partner site all listing functionality (minus the above noted) should work.
This includes, but is not limited to:
- /Listings page should load and you can see/paginate the listings
  - Appropriate listings should be shown for each user type
- Create a listing
  - All fields should be filled in and after save should still exist
- Save as draft and publish listing
- Listing status changes (draft, open, closed, etc)
- Edit listing
  - Remove fields, edit fields, add new fields, update assets (images, open houses, etc)

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
